core/modules/catalog/types/Product.ts

```diff
@@ -1,62 +1,62 @@
 export default interface Product {
   category: Array<Object>
   category_ids: Array<string>
-  climate: string
+  climate?: string
   color: string
-  color_options: Array<number>
+  color_options?: Array<number>
   configurable_children: Array<Object>
   configurable_options: Array<Object>
-  custom_attributes: any
+  custom_attributes?: any
   description: string
-  eco_collection: string
-  erin_recommends: string
-  errors: Object
+  eco_collection?: string
```

Comment:
I belive this two should be removed from the schema they’re just specific to magento demo data

Reply:
So you want me to remove eco_collection & erin_recommends from the file?